tools/icu/README.md

```diff
@@ -79,9 +75,9 @@ process.versions.icu;
 new Intl.DateTimeFormat('es', {month: 'long'}).format(new Date(9E8));
 ```
 
-(This should print your updated ICU version number, and also `January` again.)
+(This should print your updated ICU version number, and also `enero` again.)
 
-You are ready to check in the updated `deps/small-icu`. This is a big commit,
+You are ready to check in the updated `deps/full-icu`. This is a big commit,
```

Comment:
`icu-full`

Reply:
if you do `configure --with-icu-source=somewhere/ssomewhere/icu4c-62_1-src.tgz` then `deps/icu` gets created with the contents of the specified tarball.  so `deps/icu` is already taken, and expected to be transient.

We could also just call it `small-icu` - it is smaller (trimmed out unneeded stuff), just not small in data.

Reply:
Ah, in that case I'd rather rename the transient directory if that's not too much work.